CPT331.Data.Parsers/VicXmlParser.cs

```diff
@@ -0,0 +1,71 @@
+﻿#region Using References
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml;
+
+using CPT331.Core.ObjectModel;
+
+#endregion
+
+namespace CPT331.Data.Parsers
+{
+	public class VicXmlParser
+	{
+		//	Number of offences by geographic area and offence type, year ending December 2014									
+		public VicXmlParser(string filename)
+		{
+			_filename = filename;
+		}
+
+		private readonly string _filename;
+
+		public void Parse()
+		{
+			if ((String.IsNullOrEmpty(_filename) == false) && (File.Exists(_filename) == true))
+			{
+				XmlDocument xmlDocument = new XmlDocument();
+				xmlDocument.Load(_filename);
+
+				XmlNodeList xmlNodeList = xmlDocument.SelectNodes("/Workbook/Worksheet/Table/Row[position() > 1]");
+
+				//	The dataset I'm working with has a single year - 2014
+				int year = 2014;
+
+				State vicState = StateRepository.GetStateByAbbreviatedName("VIC");
+				List<LocalGovernmentArea> localGovernmentAreas = LocalGovernmentAreaRepository.GetLocalGovernmentAreasByStateID(vicState.ID);
+				List<Offence> offences = OffenceRepository.GetOffences();
+
+				List<Crime> crimes = new List<Crime>();
+
+				foreach (XmlNode xmlNode in xmlNodeList)
+				{
+					string localGovernmentAreaName = xmlNode.ChildNodes[2].InnerText.Trim();
```

Comment:
There's a nicer way to select these columns, which will be added at a later date.